services/spark-operator/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+displayName: Spark
+description: Apache Spark is a fast and general-purpose cluster computing system for big data
+category:
+  - compute
+  - bigdata
+overview: >
+  # Overview
+  Apache Spark is a fast and general-purpose cluster computing system for big data. It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. It also supports a rich set of higher-level tools including: Spark SQL for SQL and DataFrames, MLlib for machine learning, GraphX for graph processing, and Spark Streaming for stream processing.
+
```

Comment:
Do we need a richer `overview` ?

Reply:
That lgtm, but for these we can ask @mesosphere/d2iq-docs to review also!

Reply:
I don't really understand where this text is being displayed, but I guess it is in some kind of 'catalog' page in the Kommander UI?

This description seems fine, but it is pretty long, which may introduce some other concerns.   For example:

How much space is allocated 'by default' for this text?
Does the field/space/box it goes in grow or have enough room for all this to fit?
If not, does the control scroll so the user can read all of it?

I'd guess that most people looking for Spark probably know a bit about it, so I am not sure we really need a super detailed description.   If we have room for all this, then I'm ok with it, but if not, I'd note that the Spark catalog item in the old DC/OS package universe just said this:

"Apache Spark is a unified analytics engine for large-scale data processing."

https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/S/spark/900/package.json#L8

If we need to shorten this, i'd suggest just going with the first sentence you have, which is very similar:

"Apache Spark is a fast and general-purpose cluster computing system for big data."

Reply:
Yes, this is for catalog app descriptions, essentially it would look similar to this platform app details page
![image](https://user-images.githubusercontent.com/5897740/144103930-3b3d9309-f164-4d0f-9741-a4c88e26403c.png)